src/web/helpers/js/_core.js

```diff
@@ -117,6 +117,16 @@ class Core {
         return "<div class='hig__icon'>" + iconString + "</div>";
     }
 
+    _setCellWidth(width){
```

Comment:
A couple of things here. The name of this function doesn't describe what it does. Also, I don't think it has much to do with rendering components. I'd rename it to `frToPercent` and move it to `helpers/js/_frToPercent`.